### PR TITLE
nRF53 Series support for nRF BLE Controller

### PR DESCRIPTION
**Pull request by thomasstenersen**
This PR completes the support for the nRF53 Series SoC for the nRF BLE Controller.

All Bluetooth samples have been tested with the nRF BLE Controller running on the network core. The only sample that does not work out of the box is the mesh sample, which needs some general updates for nRF53 support not related to the nRF BLE Controller itself.

### CODE REVIEW

**Comment by thomasstenersen** [comment]
The CI is failing for some (of the) nrf_desktop application. Can you @pdunaj or anyone else from the nRF Desktop team help me shave off some bytes so we can get this in? :)

File: subsys/bluetooth/controller/hci_driver.c [hunk 1]
```diff
@@ -23,6 +23,12 @@
 #define BLE_CONTROLLER_IRQ_PRIO_LOW  4
 #define BLE_CONTROLLER_IRQ_PRIO_HIGH 0
 
+#if IS_ENABLED(CONFIG_SOC_SERIES_NRF52X)
+	#define BLE_CONTROLLER_PROCESS_IRQn SWI5_IRQn
+#elif IS_ENABLED(CONFIG_SOC_SERIES_NRF53X)
+	#define BLE_CONTROLLER_PROCESS_IRQn EGU0_IRQn
+#endif
```
Comment (SebastianBoe) [inline subsys/bluetooth/controller/hci_driver.c]: 51?

Should we error-out on unrecognized platforms?

Comment (SebastianBoe) [reply subsys/bluetooth/controller/hci_driver.c]: Ah, this the NRFXLIB_BT_LL ? Nevermind, sorry.

Comment (thomasstenersen) [reply subsys/bluetooth/controller/hci_driver.c]: You cannot select the controller for 51.

**Comment by thomasstenersen** [comment]
Pushed a proposal to get around the error. Allows you to set the BT_LL_NRFXLIB as the default selection for BT_LL_CHOICE, that way it won't warn if BT_LL_NRFXLIB isn't supported.

**Pull request merged**